unstructured/partition/pdf.py

```diff
@@ -363,6 +364,40 @@ def _process_pdfminer_pages(
     return elements
 
 
+def convert_pdf_to_images(
+    filename: str = "",
+    file: Optional[Union[bytes, BinaryIO, SpooledTemporaryFile]] = None,
+    chunk_size: int = 10,
+) -> Iterator[List[object]]:
```

Comment:
Should this change now?
```suggestion
) -> Iterator[PIL.Image.Image]:
```

Reply:
Fixed it. Thanks!